packages/ibm-products/src/components/Carousel/Carousel.js

```diff
@@ -164,11 +168,16 @@ export let Carousel = React.forwardRef(
         }
       }
       const outerDiv = carouselRef.current;
-      if (outerDiv) {
-        outerDiv.addEventListener('keydown', keypress);
-        return () => outerDiv.removeEventListener('keydown', keypress);
-      }
+      //if (outerDiv) {
```

Comment:
Remove if no longer needed

Reply:
Good catch... I assume its better to have the check in there, I think I was testing that and missed putting it back.